jupyterdrive/gdrive/driveutils.d.ts

```diff
@@ -0,0 +1,103 @@
+import iface = require('content-interface');
+export declare var FOLDER_MIME_TYPE: string;
+export declare var NOTEBOOK_MIMETYPE: string;
+export declare var MULTIPART_BOUNDARY: string;
+export declare enum FileType {
```

Comment:
Lots of `export declare var`s!  Why not a pattern like this : https://github.com/jdfreder/poster/blob/master/source/ts/syntax/prism.d.ts#L12